packages/angular/cli/lib/init.ts

```diff
@@ -78,11 +78,14 @@ let forceExit = false;
       console.error('Version mismatch check skipped. Unable to compare local version: ' + error);
     }
 
-    if (isGlobalGreater) {
+    const rawCommandName = process.argv[2];
+    // When using the completion command and the global version is greater,
+    // don't show the warning as otherwise this will break completion.
+    if (isGlobalGreater && rawCommandName !== 'completion') {
```

Comment:
This comment doesn't align with the if statement (should be `if (!(isGlobalGreater && rawCommandName === 'completion')` or `if (!isGlobalGreater || rawCommandName !== 'completion')`).

I think we don't want to change the existing behavior, so I think this comment should read "When using the completion command, don't show the warning as otherwise this will break completion." Is that what you're intending here?